packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -419,8 +395,6 @@ class Cursor extends SynchronousCursor implements ITreeSubscriptionCursor {
         return this.innerCursor.value;
     }
 
-    observer?: ObservingDependent | undefined;
```

Comment:
I can't add a comment on lines that aren't part of the change but is the comment on line 288 still needed?

Reply:
No, that comment should not be kept. Fixed.